### PR TITLE
Add support for -plain-http-tunneling

### PR DESCRIPTION
**Pull request by ttiurani**
Adds hoverfly-java setting related to:

https://github.com/SpectoLabs/hoverfly/pull/683

referenced in issue:

https://github.com/SpectoLabs/hoverfly/issues/650

### CODE REVIEW

File: src/main/java/io/specto/hoverfly/junit/core/HoverflyConfig.java [hunk 1]
```diff
@@ -32,6 +32,7 @@
     protected String destination;
     protected String proxyCaCert;
     protected List<String> captureHeaders;
+    protected boolean plainHttpTunneling;
```
Comment (tommysitu) [inline src/main/java/io/specto/hoverfly/junit/core/HoverflyConfig.java]: This config option is specific to local Hoverfly only, it should be moved to `LocalHoverflyConfig`.

Comment (ttiurani) [reply src/main/java/io/specto/hoverfly/junit/core/HoverflyConfig.java]: Ah, right. Moved it there now.

**Comment by tommysitu** [comment]
I've added the Reactor Netty test as well. It looks good. Thank you!

**Comment by tommysitu** [comment]
The release should be available later today, which also address an issue for Windows users.

**Comment by ttiurani** [comment]
@tommysitu Thanks for this, I tested that also the release works great. I didn't give up on this issue, because Hoverfly is really a the best tool for service virtualization. Hopefully you can keep on updating it and pushing to v1.0! I'll be sure to report any other issues we come across.

**Comment by tommysitu** [comment]
Thanks for your feedback!

**Pull request merged**